riocli/v2client/client.py

```diff
@@ -21,6 +21,7 @@
 
 import click
 import magic
+from docutils.nodes import header
```

Comment:
todo: you can remove this. I think this may have been imported by mistake by PyCharm.

Reply:
Hi @pallabpain , this is done.